components/Select/Select.tsx

```diff
@@ -235,7 +235,7 @@ export const Select = React.forwardRef<React.ElementRef<typeof StyledSelect>, Se
         disabled={props.disabled}
       >
         <StyledSelect ref={forwardedRef} {...props} />
-        <StyledCaretSortIcon size={size} />
+        <StyledCaretSortIcon size={size} cursor={cursor} />
```

Comment:
@matthieuh I binded the `cursor` prop from `Select` down to `StyledCaretSortIcon`, as it has an unused `cursor` variant.

Do you think it's useful? Should I instead remove the variant from the subcomponent?